love/src/components/MainTel/CableWraps/CableWraps.module.css

```diff
@@ -49,14 +49,24 @@
 
 .divCameraWrap {
     margin-top: 2em;
+    display: inline-block;
+    height: 70%;
+    width: 100%;
+	vertical-align: middle; 
```

Comment:
Are you using pre-commit for linting and formatting? this line should have been reviewed by ESLint and formatted by Prettier